src/site/content/en/blog/lighthouse-ci/index.md

```diff
@@ -0,0 +1,351 @@
+
+<figure class="w-figure">
+  <img src="./github-action2.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>
+
+You can navigate to the Lighthouse report corresponding to a particular commit from the "Actions" tab. Click on the commit, click on the "Lighthouse CI" workflow step, then expand the results of the "run Lighthouse CI" step.
+
+<figure class="w-figure">
+  <img src="./github-action3.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>
+
+
+You've just set up a GitHub Action to run Lighthouse CI. This will be most useful when used in conjunction with a GitHub [status check](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks). 
+
+A status check, if configured, is a message that appears on every PR and typically includes information such as the results of a test or the success of a build.
+
+<figure class="w-figure">
+  <img src="./github-action1.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>)
+
+The steps below explain how to set up a status check for Lighthouse CI.
+
+1.  Go to the [Lighthouse CI GitHub App page](https://github.com/apps/lighthouse-ci) and click **Configure**.
+2.  (Optional) If you're part of multiple organizations on GitHub, choose the organization that owns the repository for which you want to use Lighthouse CI.
+3.  Select **All repositories** if you want to enable Lighthouse CI in all repositories or select **Only select repositories** if you only want to use it in specific repositories, and then select the repositories. Then click **Install & Authorize**.
+4.  Copy the token that is displayed. You'll use it in the next step.
+5.  To add the token, navigate to the **Settings** page of your GitHub repository, click **Secrets**, then click **Add a new secret**. 
+
+<figure class="w-figure">
+  <img src="./github-action4.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>
+
+
+6.  Set the **Name** field to `LHCI_GITHUB_APP_TOKEN` and set the **Value** field to the token that you copied in the last step and then click the **Add secret** button.
+7.  The status check is ready for use. To test it, [create a new pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) or push a commit to an existing pull request.
+
+
+## Setup the Lighthouse CI Server
+
+The Lighthouse CI server provides a dashboard for exploring historical Lighthouse reporting. It can also act as a private, long-term datastore for Lighthouse reports.
+
+<figure class="w-figure">
+  <img src="./server1.png" alt="Screenshot of the Lighthouse CI Server dashboard" class="w-screenshot">
+</figure>
+
+<figure class="w-figure">
+  <img src="./server2.png" alt="Screenshot of comparing two Lighthouse reports in Lighthouse CI Server" class="w-screenshot">
+</figure>
```

Comment:
Please replace the hardcoded text with numbers.

![image](https://user-images.githubusercontent.com/4713486/88339357-bfc71700-ccee-11ea-9099-90beb886a242.png)

For example, the top arrow should point to the number `1`. The second-from-top would be pointing to `2`, etc.

Below the image you create an ordered list that explains `1`, `2`, etc.

This makes the content more resilient to language translations